wpengine/lostromos
# WIP: Remove test splits for unit and integration.

Signed-off-by: Derek Rushing <derek.rushing21@gmail.com>

# What Are We Doing Here

1. Adding wpdrush21 to the codeowners for the .travis.yml file.
2. Removing ADRS from the codeowners file since they aren't used.
3. Remove split for integration and unit tests since we aren't seeing real speed gains.

## How to Verify

This is really only verifiable through Travis, so just need to let it run and see.
